### PR TITLE
[ADD] report for payment order

### PR DESCRIPTION
**Pull request by GillesTephaneMeyomesse**
Adding Qweb report for Payment order

### CODE REVIEW

File: account_payment_order/data/account_payment_order_printing.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <!-- QWeb Report -->
+        <report
+            id="action_print_payment_order"
+            model="account.payment.order"
+            string="Payment Order"
+            report_type="qweb-pdf"
+            name="account_payment_order.print_account_payment_order_main"
+            file="account_payment_order.print_account_payment_order_main"
+            attachment_use="True"
+            attachment="((object.name or '').replace('/','')+'.pdf')"
```
Comment (adrienpeiffer) [inline account_payment_order/data/account_payment_order_printing.xml]: I think it's not to use and store report as attachment

File: account_payment_order/README.rst [hunk 2]
```diff
@@ -32,6 +32,8 @@ You can create a Debit Order via the menu Accounting > Payments > Debit Orders a
 
 This module also adds a button *Add to Payment Order* on supplier invoices and a button *Add to Debit Order* on customer invoices.
 
+You can print a Payment Order via the menu Accounting > Payments > Payment Orders and then select the move lines to print.
```
Comment (adrienpeiffer) [inline account_payment_order/README.rst]: I don't understand the meaning about selecting moves lines

File: account_payment_order/report/print_account_payment_order.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                    <p t-field="doc.payment_mode_id.name"/>
+                </div>
+                <div t-if="doc.name" class="col-xs-2">
+                    <strong>Reference</strong>
+                    <p t-field="doc.name"/>
+                </div>
+                <div t-if="doc.journal_id.bank_id.id" class="col-xs-2">
+                    <strong>Used Account:</strong>
+                    <p>
+                        <span t-raw="'%s &lt;br&gt;' % doc.journal_id.bank_id.name if doc.journal_id.bank_id.name else ''"/>
```
Comment (adrienpeiffer) [inline account_payment_order/report/print_account_payment_order.xml]: Actuallly, there is a field with the company bank account on payment order

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

File: account_payment_order/data/account_payment_order_printing.xml (hunk 1 above)
Comment (GillesTephaneMeyomesse) [reply account_payment_order/data/account_payment_order_printing.xml]: Thanks for review

**Comment by adrienpeiffer** [comment]
@GillesTephaneMeyomesse Could you sqash your last commit ?

**Comment by GillesTephaneMeyomesse** [comment]
Hi @adrienpeiffer last commit is squashed

**Review by adrienpeiffer — APPROVED** [review]
:+1: (Code review and test)

File: account_payment_order/data/account_payment_order_printing.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
```
Comment (pedrobaeza) [inline account_payment_order/data/account_payment_order_printing.xml]: Shouldn't this be in `report` folder?

**Comment by alexis-via** [comment]
Great initiative.

In the report, I would have looped on bank payment lines instead of payment lines, because bank payment lines is what is really used in the bank payment file, so it's what is executed by the bank. What did you choose to loop on payment lines ?

I would also call the report "Payment/Debit Order" instead of just "Payment Order", because it is also available for Debit Orders.

**Comment by sbidoul** [comment]
@alexis-via regarding payment lines vs bank lines, it depends on the purpose of the report. If the goal is to send it to the bank, then what you propose may be better. If the goal is to make is to show it to a manager for validation, looping on payment lines is more useful.

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by adrienpeiffer — APPROVED** [review]
:+1: again

**Review by benwillig — APPROVED** [review]

**Pull request merged**